daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -30,8 +30,9 @@ dapr run [flags] [command]
 | `--app-ssl`                    |                      | `false`                                                                            | Enable https when Dapr invokes the application                                                       |
 | `--components-path`, `-d`      |                      | Linux/Mac: `$HOME/.dapr/components` <br/>Windows: `%USERPROFILE%\.dapr\components`   | The path for components directory                                                                  |
 | `--config`, `-c`               |                      | Linux/Mac: `$HOME/.dapr/config.yaml` <br/>Windows: `%USERPROFILE%\.dapr\config.yaml` | Dapr configuration file                                                                            |
-| `--dapr-grpc-port`             | `DAPR_GRPC_PORT`     | `50001`                                                                            | The gRPC port for Dapr to listen on                                                                  |
-| `--dapr-http-port`             | `DAPR_HTTP_PORT`     | `3500`                                                                             | The HTTP port for Dapr to listen on                                                                  |
+| `--dapr-grpc-port`, `-G`       | `DAPR_GRPC_PORT`     | `50001`                                                                            | The gRPC port for Dapr to listen on                                                                  |
+| `--dapr-internal-grpc-port`, `-I` |                      | `50002`                                                                            | The gRPC port for the Dapr Internal API to listen on                                                         |
```

Comment:
```suggestion
| `--dapr-internal-grpc-port`, `-I` |                      | `50002`                                                                            | gRPC port for the Dapr Internal API to listen on; used when parsing the value from a local DNS component                                                        |
```

Grabbed this from dapr/cli#876 ....does this work? @msfussell

Reply:
@msfussell thoughts?

Reply:
@msfussell friendly bump

Reply:
Yes that link also needs to also be updated. And internal should be lower case.
```suggestion
| `--dapr-internal-grpc-port`, `-I` |                      | `50002`                                                                            | The gRPC port for the Dapr internal API to listen on                                                         |
```
This "; used when parsing the value from a local DNS component" still does not tell me why I would use this. Why would I possible ever change this to and when you say DNS component, are you referring the Name Resolution component?

Reply:
@msfussell @shubham1172 does this help clarify that?

```suggestion
| `--dapr-internal-grpc-port`, `-I` |                      | `50002`                                                                            | The gRPC port for the Dapr internal API to listen on. Set during development for apps experiencing temporary errors with service invocation failure or going through a firewall. Can be any value greater than 1024 and must be different for each app.              |
```

Reply:
> "Set during development for apps experiencing temporary errors with service invocation failure due to mDNS caching"...

Reply:
@shubham1172 updated my suggestion ^^